ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -31,20 +40,31 @@ check_pull_number() {
 queue_url=$(cat "/var/run/token/dpu-token/queue-url")
 queue_endpoint=$(cat "/var/run/token/dpu-token/queue-endpoint")
 endpoint_resolve="${queue_endpoint}:80:10.30.45.137"
-queue_put_response=$(trigger_build)
 
-if [ $? -ne 0 ]; then
-    echoerr "Error: Failed to trigger build."
-    exit 1
-fi
+#Check timeout wa
+timed_out_response=$(check_timeout_status)
+
+check_timed_out_return_code=$(echo "$timed_out_response" | jq -r '.return_code')
+
+if [ "$check_timed_out_return_code" -eq 400 ]; then
```

Comment:
Can we remove the nesting? It makes it harder to tell exactly what we are doing here? What does a timeout return code of 400 mean? A comment would be helpful